command/seal_migration_test.go

```diff
@@ -468,6 +446,190 @@ func testSealMigrationTransitToTestSeal(t *testing.T, setup teststorage.ClusterS
 	}
 	verifyBarrierConfig(t, b, wrapping.Test, 1, 1, 1)
 	verifyBarrierConfig(t, r, wrapping.Shamir, 5, 3, 0)
+
+	// Now that migration is done, we can stop the transit cluster, since we
+	// can seal/unseal without it.
+	tcluster.Cleanup()
+	tcluster = nil
+
+	if err := client.Sys().Seal(); err != nil {
```

Comment:
You could use the helpers here, e.g.

```
cluster.BarrierKeys = cluster.RecoveryKeys
testhelpers.EnsureCoresSealed(t, cluster)
testhelpers.EnsureCoresUnsealed(t,cluster)
```

This would have the benefit of checking perf standbys too.

Reply:
I like this idea, but I want to save it for a separate cleanup PR, since we really ought to do it everywhere in `seal_migration_test`.